src/views/Home.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <AboutContent v-if="shouldShowAbout" />
   <SspContent v-else-if="shouldShowSSP" />
-  <b-container v-else="shouldShowResults">
+  <b-container v-else-if="shouldShowResults">
```

Comment:
Please review this video as to how this could be a bit cleaner overall: https://www.youtube.com/watch?v=KS4eizPXRCQ